Bool Watering
!StartTime
!FinishedTime
!ErrorNr